File: Unidad 5. AWT y Swing/Clase10/src/clase10/Persona.java
package clase10;

public class Persona {

    private String codigo;
    private String nombre;
    private int edad;
    private double tamaño;

    public Persona(String codigo, String nombre, int edad, double tamaño) {
        this.codigo = codigo;
        this.nombre = nombre;
        this.edad = edad;
        this.tamaño = tamaño;
    }

    public void Presentarme(){
        System.out.println("-------------------------------");
        System.out.println("Hola, me llamo: " + nombre);
        System.out.println("Mi codigo es: " + codigo);
        System.out.println("-------------------------------");
    }
    
    public String getCodigo() {
        return codigo;
    }

    public void setCodigo(String codigo) {
        this.codigo = codigo;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public int getEdad() {
        return edad;
    }

    public void setEdad(int edad) {
        this.edad = edad;
    }

    public double getTamaño() {
        return tamaño;
    }

    public void setTamaño(double tamaño) {
        this.tamaño = tamaño;
    }
}

File: Unidad 7. Programacion Web/Backend/Persona.py
#PARA NO PERDER LA REFERENCIA A JAVA USAMOS CLASES
class Persona:
    # NUESTRO NUEVO CONSTRUCTO
    # __init__ sera nuestro metodo que se ejecuta al crear un objeto
    # self, hace referencia al objeto actual    
    def __init__(self,nombre,apellido,edad):
        self.nombre = nombre
        self.apellido = apellido
        self.edad = edad

    # METODOS GET
    # Creamos nuestros metodos para obtener la informacion, usando self
    def getNombre(self):
        return self.nombre
    
    def getApellido(self):
        return self.apellido
    
    def getEdad(self):
        return self.edad

    # METODOS SET
    # Creamos nuestros metodos para setear la informacion, usando el self y el parametro
    def setNombre(self, nombre):
        self.nombre = nombre
    
    def setApellido(self, apellido):
        self.apellido = apellido
    
    def setEdad(self, edad):
        self.edad = edad
